scripts/gitops-deploy.sh

```diff
@@ -52,5 +52,6 @@ for part in operators components; do
     done
     rm -rf "${OUTPUT_DIR}/${part}/kustomization.yaml"
 done
+touch "${UC_DEPLOY}/helm-configs/${DEPLOY_NAME}/keystone.yaml"
```

Comment:
I think it may be slightly unclear what type of customisation is available - should we add some comment in that file instead? 
for example

```bash
echo "# Add your custom openstack-helm values here" > "${UC_DEPLOY}/helm-configs/${DEPLOY_NAME}/keystone.yaml"

Reply:
Sure I can do that.

Reply:
I expanded on that a bit. Lemme know if that change makes sense.